airbyte-webapp/src/components/JobItem/components/AttemptDetails.tsx

```diff
@@ -14,11 +15,7 @@ interface AttemptDetailsProps {
   hasMultipleAttempts?: boolean;
 }
 
-const getFailureFromAttempt = (attempt: AttemptRead) => {
-  return attempt.failureSummary && attempt.failureSummary.failures[0];
-};
-
-const AttemptDetails: React.FC<AttemptDetailsProps> = ({ attempt, className, hasMultipleAttempts }) => {
+export const AttemptDetails: React.FC<AttemptDetailsProps> = ({ attempt, className, hasMultipleAttempts }) => {
   const { formatMessage } = useIntl();
 
   if (attempt.status !== AttemptStatus.succeeded && attempt.status !== AttemptStatus.failed) {
```

Comment:
It just needs to be cleaned up because of changes that were made to get rid of the Status enum which was tracking too many statuses:
https://github.com/airbytehq/airbyte/pull/19599/files#diff-9438e96560c48e89bef8ea05895b60ab0cc824a87ae8408196d4de80a12b4f4aL24